kapitan/cli.py

```diff
@@ -429,20 +436,32 @@ def secret_write(args, ref_controller):
         tag = '?{{awskms:{}}}'.format(token_path)
         ref_controller[tag] = secret_obj
 
-    elif token_name.startswith("ref:"):
+    elif token_name.startswith("base64:"):
```

Comment:
This will break secret parding for all old formats right? Woukd it make sense to support the old type here too? Maybe at least print a warning message for the user so they know what to do/how to upgrade?

Reply:
I'm thinking it might be better to provide a migration script. Sure, we could support `ref` but also other things will change and break, like renaming `--secrets-path` into `--refs-path` and the default value from `./secrets/` to `./refs/`. So I believe it's wiser to break and provide a clear migration script.